app/helpers/agenda-name.js

```diff
@@ -0,0 +1,11 @@
+import { helper } from '@ember/component/helper';
+
+export default helper(async function agendaName([agenda]) {
```

Comment:
Wondering whether templates automatically update (without the need for the `await` helper) when defining `agendaName` as an (async) getter on the `agenda` model like
```javascript
get agendaName() {
  let prefix = this.status.get('isDesignAgenda') ? 'Ontwerpagenda' : 'Agenda';
  let name = this.serialNumber || '';
  return `${prefix} ${name}`.trim();
}
```

I know it's an anti-pattern we would like to avoid, but this case seems 'legimate' to me. It's a very simple getter, it doesn't introduce additional network requests compared to the helper and it makes the template simpler since there is no need to use `await`.